concepts/vocabulary.md

```diff
@@ -0,0 +1,31 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Vocabulary
+description: The set of supported words
+---
+
+The **vocabulary** of a model is the set of supported words.
+In a machine translation system, the vocabulary may refer to the set of words supported in either the input or the output.
+A word that's supported in the output is a word that can be generated by the translation model.
+A word that's supported in the input is a word that can be translated.
```

Comment:
Shall we change the order and first define `input-supported word` and second `output-supported word`? I know it's a tiny edit, but it would follow the order introduced by the previous sentence.